mv: cannot stat 'job_logs/train.err': No such file or directory
Starting training on cx3-20-0.cx3.hpc.ic.ac.uk at Fri 18 Apr 15:43:28 BST 2025
Using config: config_template.json
Traceback (most recent call last):
  File "/rds/general/user/psp20/home/Brain-Network/source/simple_cnn.py", line 289, in main
    config, verbose, debug = parse_args()
                             ^^^^^^^^^^^^
  File "/rds/general/user/psp20/home/Brain-Network/source/simple_cnn.py", line 277, in parse_args
    with open(args.config, 'r') as f:
         ^^^^^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: 'config_template.json'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/rds/general/user/psp20/home/Brain-Network/source/simple_cnn.py", line 423, in <module>
    main()
  File "/rds/general/user/psp20/home/Brain-Network/source/simple_cnn.py", line 418, in main
    logger.exception("Error occured")
    ^^^^^^
UnboundLocalError: cannot access local variable 'logger' where it is not associated with a value
Finished at Fri 18 Apr 15:44:19 BST 2025

====================================
CPU Time used: 00:00:01
CPU Percent: 8%
Memory usage: 204476kb
Approx Power usage: 0.0
Walltime usage: 00:01:02

====================================
